Modules/Blueprints/include/selxBlueprint.h

```diff
@@ -103,10 +104,13 @@ class Blueprint : public itk::DataObject
   void MergeFromFile(const std::string& filename);
 
   void SetLogger( Logger::Pointer logger );
+
 private:
 
-  BlueprintImplPointer m_Blueprint;
+  // The order of the logger and the blueprint matters, since the lifetime of the logger should always exceed that of the blueprint.
   Logger::Pointer m_Logger;
+  BlueprintImplPointer m_Blueprint;
```

Comment:
Maybe name should be m_BlueprintImpl?

Reply:
Yes, that would be better. I see that Coert proposes to call it m_Implementation in #147.